misk/src/main/kotlin/misk/web/mediatype/MediaTypes.kt

```diff
@@ -0,0 +1,17 @@
+package misk.web.mediatype
+
+import okhttp3.MediaType
+
+object MediaTypes {
+    const val APPLICATION_JSON = "application/json;charset=utf-8"
```

Comment:
internal or private ?

Reply:
It's intended for use by applications, so neither?

Reply:
Ahh, cause the annotation can’t take a MediaType, it needs to take a string. Yep